src/extensions/filter-control/bootstrap-table-filter-control.js

```diff
@@ -624,9 +625,26 @@ $.extend($.fn.bootstrapTable.Constructor.EVENTS, {
 $.extend($.fn.bootstrapTable.defaults.icons, {
   clear: {
     bootstrap3: 'glyphicon-trash icon-clear'
-  }[$.fn.bootstrapTable.theme] || 'fa-trash'
+  }[$.fn.bootstrapTable.theme] || 'fa-trash',
+  filterControlSwitchHide: {
+    bootstrap3: 'glyphicon-zoom-out icon-zoom-out'
```

Comment:
Icons for the other Themes are missing e.g. https://live.bootstrap-table.com/code/UtechtDustin/3488

Reply:
I think it only fails for the Materialize theme. I copied the approach from other extensions and indeed they also show don't show an icon for that theme (e.g. [print](https://live.bootstrap-table.com/code/marceloverdijk/3491)).

I've defined icons for the Materialize theme as well now. See [demo](https://live.bootstrap-table.com/code/marceloverdijk/3492).